passport-anonymous/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for passport-anonymous 1.0
+// Project: https://github.com/jaredhanson/passport-anonymous
+// Definitions by: Pavel Puchkov <https://github.com/0x6368656174>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import passport = require("passport");
+
+declare class Strategy implements passport.Strategy {
```

Comment:
this does not look accurate.. see https://github.com/jaredhanson/passport-anonymous/blob/master/lib/index.js#L15


the module should have `export = Strategy` at the bottom, and `Strategy` should have a static member `static Strategy: typeof Strategy`